### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by shrushti2000**
### added wishlist page

image

### CODE REVIEW

File: pages/wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../../style.css">
+</head>
+
+<body>
+    <div class="navigation-container flex-hz">
```
Comment (SRUJANA-PENUGONDA13) [inline pages/wishlist/wishlist.html:13]: For navigation you followed one naming convention where as for card you followed other. Try to be consistent.

File: pages/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        <div class="navigation-item-3 flex-hz">
+            <button class="btn btn-primary">Logout</button>
+            <i class="fa fa-shopping-cart nav-icons"><span class="badge-w-icon">1</span></i>
+            <i class="fa fa-heart nav-icons"><span class="badge-w-icon">10</span></i>
+        </div>
+    </div>
+    <div class="product-display-container">
+        <h2 class="sub-heading">My WishList</h2>
+        <div class="product-container">
+            <div class="card__container flex-vt">
```
Comment (SRUJANA-PENUGONDA13) [inline pages/wishlist/wishlist.html:32]: As mentioned try to follow consistent naming standards.

**Comment by SRUJANA-PENUGONDA13** [comment]
I felt you can add Wishlist icon on top right corner of the card so that it will be easy to access. Overall it's looks good

**Review by HarshnaNHaswani — COMMENTED** [review]
Looks good to me. Few suggestions
- keep logout after icons in nav
- add more spacing between icons and button, so that they seem separate
- don't mix icon-button and normal button in card.
- if you want ❤️ in card then change its location

**Pull request merged**